cluster/cluster.go

```diff
@@ -545,6 +565,10 @@ func (p *Peer) Name() string {
 	return p.mlist.LocalNode().Name
 }
 
+func (p *Peer) Address() string {
```

Comment:
Who needs / uses this?

Reply:
This is used by the API to expose the node IP, but also internally to manage the peers as they join/leave/reconnect.

I have added a comment to signal what this returns.

Reply:
Now I realise what you meant, the actual address is always used from the `Peers()` which returns the `Node` it is not needed in the base peer. Great catch!